app/src/main/java/dev/lfspersson/arctouchtmdb/network/RestService.java

```diff
@@ -0,0 +1,33 @@
+package dev.lfspersson.arctouchtmdb.network;
+
+import dev.lfspersson.arctouchtmdb.BuildConfig;
+import dev.lfspersson.arctouchtmdb.database.models.DiscoverModel;
+import dev.lfspersson.arctouchtmdb.database.models.GenreListModel;
+import retrofit.Call;
+import retrofit.GsonConverterFactory;
+import retrofit.Retrofit;
+import retrofit.http.GET;
+import retrofit.http.Query;
+
+/**
+ * Created by LFSPersson on 30/11/16.
+ */
+
+public interface RestService {
+    @GET("discover/movie")
```

Comment:
Both endpoints has the same data structure model so I decided to use Discover endpoint because sorting criteria is more attractive than the Upcoming endpoint in my opinion.